cross-functional/workflow.md

```diff
@@ -1,34 +1,125 @@
+  - Investment initiatives are an investment in our team, technology, etc that does not immediately deliver value to communities, but helps us build a better foundation to do so.
+- **Task lists**: Are specific actions or deliverables that are taken to complete an initiative. These are defined as `[tasklist]` objects in each initiative, and are the main units of work that drive our actions.
+
+````{admonition} Example structure of an initiative
+:class: dropdown
+Here's the rough structure of an initiative, use this to guide the creation of new initiatives.
+
+```markdown
+title: Initiative name in GitHub issue
+
+### Description
+
+A short description of the initiative, with enough context so that people can understand
+what kind of work it will entail, and why it is important to 2i2c's overall goals.
+
+### Linked goal or project
+
+- Link to a strategic goal or a project (2i2c-based, upstream, etc)
+
+### Why is this important?
+
+- This will allow us to XXX
+
+### Measures and definition of done
 
-The [`#solution-forum` Slack channel](https://2i2c.slack.com/archives/C06G5FAAT63) is a space for cross-organization initiatives and strategic conversation.
+- We expect to see XXX
+- This will be complete when YYY
 
-- It is meant for discussing organization-wide practices, policies, and planning.
-- It will have a combination of strategic- and tactical-level discussion. (in the future we may split this into two channels if it's too much)
-- It will initially focus its discussion around our "cross-functional initiatives" board and the items in it.
-- Anyone that participates in conversations in this channel should dedicate the attention to understand the context for discussion and participate until topics are resolved.
-- Area leads are expected to monitor this channel and participate in cross-team conversations as they arise.
-- All 2i2c staff have visibility into the channel and may participate in conversations, but are not expected to do so as part of their daily responsibilities.
+```[tasklist]
+- [ ] A growing list of tasks and actions used to coordinate work around this initiative.
+- [ ] Each should be an actionable issue that we track independently.
+- [ ] This may be empty when the initiative is created! The initiative's goals should be used to decide what action issues to create next.
+```
+````
 
-### Team members expected to monitor this channel
+### Bi-weekly initiatives meetings
 
-Below is a list of the team members that we **expect** to participate in this channel on a regular basis:
+There are two bi-weekly meetings with the initiatives team.
 
+- **Initiative planning and refinement**: A combindation of **coordination** and **prioritization**. We first walk the board and ensure each initiative has clear next steps, assigned people, and that any coordination between teams is clear. We next zoom out and discuss our overall portfolio of initiatives to ensure we're sequencing them properly.
```

Comment:
Combining 'refinement' and 'planning' into a single ceremony might be a challenging in practice. If I had to choose one, I'd suggest that the planning (prioritization) conversation is the more important part to do synchronously in a bi-weekly meeting. 

Initiatives of course need refinement but I don't think it makes sense to have all team members participate in a refinement of all initiatives.  Even coordination may not be appropriate in time-bounded meeting.  I understand it is useful to have a time to figure out coordination issues or evaluate next steps but I think we should flip the sequence around: start by zooming out at having the prioritization/planning discussion and then, if there is time, discuss individual initiatives that need attention.  In particular, I don't think we want to 'walk the board' and discuss every initiative that is work-in-process.

Reply:
I don't feel like I have enough experience or information to have strong opinions here. I'd suggest that we see how the first few meetings go, and decide if / how we want them to be useful (e.g. as a high level prioritization effort, or a more tactical coordination effort). My feeling is that we want to use these meetings to trigger cross-team conversations to ensure that we are splitting work across our teams effectively and providing clarity about what comes next in our sprint planning.